### PR TITLE
update list of board members

### PR DESCRIPTION
**Pull request by davidchambers**
Welcome @vsmart! :tada:

Thank you for your contributions @annapawlicka and @Malwine. :)

Franka, let me know how you'd like your name to appear in this document. You needn't provide a Nachname if you'd prefer not to do so.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,10 +4,9 @@ This repository saves and keeps tracking various documents or etc.
 
 #### Who is the board
 
-- Anna Pawlicka [@annapawlicka](https://github.com/annapawlicka)
 - Chris Koehnen [@crkoehnen](https://github.com/crkoehnen)
 - David Chambers [@davidchambers](https://github.com/davidchambers)
-- Malwine Gier [@Malwine](https://github.com/Malwine)
+- Franka [@vsmart](https://github.com/vsmart)
```
Comment (davidchambers) [inline README.md]: Would you like to be listed simply as Franka, @vsmart?

Comment (vsmart) [reply README.md]: Hi sorry for the late reply! Could you list me as Franziska Schmidt? Thanks! :)

Comment (davidchambers) [reply README.md]: I've updated the pull request. Please confirm that your details are correct, and merge the pull request if they are. :)

**Comment by vsmart** [comment]
Looks great! thanks @davidchambers!

excited spongebob

**Pull request merged**